src/cli/remove.rs

```diff
@@ -34,6 +35,10 @@ pub struct Args {
     /// The platform for which the dependency should be removed
     #[arg(long, short)]
     pub platform: Option<Platform>,
+
+    /// The feature for which the dependency should be removed
+    #[arg(long)]
+    pub feature: Option<FeatureName>,
```

Comment:
I think this should just be an Option<String>, None is FeatureName::Default Some is a named Feature

Reply:
I'm not sure what you mean with your comment?

Reply:
If you specify nothing you are referring to the "default" feature. 
If you specify a name you are always referring to something `FeatureName::Named`.

So in that sense the type of `feature` here could just be `Option<String>`. You can get the feature name through:

```rust
feature.map_or(FeatureName::default, FeatureName::Named)
```

Reply:
Got it, fix in : https://github.com/prefix-dev/pixi/pull/680/commits/b30170d86ee26e72e87280f4c026f68ab9f120f5